### PR TITLE
Make initial revisions for PLOS one.

### PR DESCRIPTION
**Pull request by drvinceknight**
This includes:

- The `response/main.md` 
- All changes to the manuscript (they are minor)

Still to do: add the correct axis to the plot (point 4 of the second reviewer).

### CODE REVIEW

File: main.tex [hunk 1]
```diff
@@ -392,18 +392,20 @@ \subsection{Measuring Extortion from the History of Play}
 becomes:
 
 \begin{equation}
-    -x^*_2 < x^*_1 \label{eqn:measured_condition_for_chi}
+    \frac{-x^*_2}{x^*_1} > 1
```
Comment (marcharper) [inline main.tex]: Was this better before, since the numerator could be zero?

File: main.tex [hunk 2]
```diff
@@ -457,6 +459,8 @@ \subsection{Validation}
     \includegraphics[width=.8\textwidth]{./assets/img/sserror_in_stewart_plotkin/main.pdf}
     \caption{\(\SSe\) and best fitting \(\chi\) for~\cite{Stewart2012},
         ordered both by number of wins and overall score.
+        A win is when a strategy obtains a higher score in than the player it is
```
Comment (marcharper) [inline main.tex]: remove 'in'

File: main.tex [hunk 3]
```diff
@@ -666,12 +672,15 @@ \subsubsection{Replicator Dynamics}
 
 \subsubsection{Finite Population Dynamics: Moran Process}
 
+The Moran Process is an evolutionary model of evolutionary in a finite
+population. Of specific interest is the probability probability of a single
```
Comment (marcharper) [inline main.tex]: remove one probability

**Review by marcharper — APPROVED** [review]

File: main.tex (hunk 1 above)
Comment (drvinceknight) [reply main.tex]: I'm not actually sure. As it was before is technically not correct when $x_1^*$ is negative. I believe $\chi$ itself is not defined for $x_1^*=0$. I'll add a note about that. Let me know what you think though.

Comment (marcharper) [reply main.tex]: hmm yeah $\alpha=0$ or $\beta=0$ are special/unusual cases.

Comment (marcharper) [reply main.tex]: Maybe $\frac{-x^{*}_{2}{x^{*}_{1}} = \chi > 1$ more clearly connects the $x$ values, $\chi$, $\alpha$, and $\beta$  (github doesn't seem to render the *)

Comment (drvinceknight) [reply main.tex]: Yeah good idea, that is better.

Comment (drvinceknight) [reply main.tex]: [5256bed](https://github.com/drvinceknight/testing_for_ZD/pull/133/commits/5256bed5ea937360d7a7a34bf5bb86382998ebf4)

File: response/main.md [hunk 4]
```diff
@@ -0,0 +1,151 @@
...
+> If yes, please provide a detailed list of your revisions so I can check it.
+
+We agree that the comments by the handling editor at the time were reasonable.
+We took on their comments and made all the required changes.
+
+The comments from the area editor were:
+
+- It is not clear from Figure 1 whether sorting is made in an ascending or descending order from left to right. Readers have to guess it.
+
+This has been clarified in the caption.
```
Comment (Nikoleta-v3) [inline response/main.md:20]: I don't think this is clarified in the caption. You can figure out the order by reading the Figure discussion, but we're not explicitly stating it.

File: main.tex [hunk 5]
```diff
@@ -505,7 +509,7 @@ \subsection{Numerical experiments}
 \begin{figure}[!htbp]
     \centering
     \includegraphics[width=.95\textwidth]{./assets/img/sse_chi_probabilities_in_full/main.pdf}
-    \caption{\(\SSe\) and \(P(DD)\) and state probabilities for the strategies for
+    \caption{\(\SSe\) and probability of mutual defection (\(P(DD)\)) and state probabilities for the strategies for
```
Comment (Nikoleta-v3) [inline main.tex]: I'm sorry if I'm being silly here, but what does `and state probabilities for the strategies` refer to?

File: response/main.md [hunk 6]
```diff
@@ -0,0 +1,151 @@
...
+- Section 1, 2nd last paragraph: "in to" -> "into"
+- Section 2, 2nd paragraph: "by elements of R^4 mapping" -> "by elements of R^4, mapping"
+- Below Eq.(6) "In Section, ": a section number is missing.
+
+These have all been addressed.
+
+- Below Eq.(15) "triangular (15) plane (13) in 3 dimensions (14)":
+  This is not a reader-friendly way of citing equations.
+
+We do not entirely understand what was implied here. We feel that having the
```
Comment (Nikoleta-v3) [inline response/main.md]: If we decide to re-write just to make the reviewer happy,  here's a potential suggestion. Feel free to ignore. 

`Based on Equations (13-15), it is evident that all extortionate strategies reside on a triangular plane within a three-dimensional space.`

**Review by Nikoleta-v3 — APPROVED** [review]
Thank you very much for your work, @drvinceknight ! I just have one comment; the others are a question and a suggestion you can ignore.

Also would it be okay to add my current affiliation as well?

`Max Planck Research Group on the Dynamics of Social Behavior, Max Planck Institute for Evolutionary Biology, Plo ̈n, Germany`

File: response/main.md (hunk 4 above)
Comment (drvinceknight) [reply response/main.md:20]: You're right! I thought I'd added it in. Good catch.

File: main.tex (hunk 5 above)
Comment (drvinceknight) [reply main.tex]: 🤔I have no idea! 🫤

Have removed it.

File: response/main.md (hunk 6 above)
Comment (drvinceknight) [reply response/main.md]: That's nice! I'll do that.

**Comment by drvinceknight** [comment]
I've made these changes, if you get a chance @Nikoleta-v3 I'd appreciate you making sure I've made those changes correctly. I'll be submitting this afternoon (the deadline is tomorrow) so no trouble if you don't have time.

**Comment by Nikoleta-v3** [comment]
All looks good to me. Thank you! 😄

**Pull request merged**